packages/calcite-components/src/components/stepper/stepper.e2e.ts

```diff
@@ -739,23 +771,26 @@ describe("calcite-stepper", () => {
     expect(await stepperItem3.getProperty("selected")).not.toBe(true);
   });
 
-  it("should select the next enabled stepper-item if first stepper-item is disabled", async () => {
-    const page = await newE2EPage();
-    await page.setContent(
-      html`<calcite-stepper>
-        <calcite-stepper-item heading="Step 1" id="step-1" disabled>
-          <div>Step 1 content</div>
-        </calcite-stepper-item>
-        <calcite-stepper-item heading="Step 2" id="step-2">
-          <div>Step 2 content</div>
-        </calcite-stepper-item>
-      </calcite-stepper>`,
-    );
+  it.each(["disabled", "hidden"])(
+    "should select the next enabled stepper-item if first stepper-item is %s",
+    async (accessibility) => {
```

Comment:
IMO, `accessibility` seems out of place here. Would `stateProp` be a better fit?